### DISCUSSION TITLE
graphql/codegen/docs/guides/flutter-freezed

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#10193] [Docs Discussions]
# graphql/codegen/docs/guides/flutter-freezed

Boost 🚀 your development of Dart/Flutter apps using a GraphQL API by generating models directly from your GraphQL Schema.

https://the-guild.dev/graphql/codegen/docs/guides/flutter-freezed

**Comment by Parables** [comment]
Hello there, I am Parables, the author of this plugin.

I am glad you've found this plugin and I hope it helps you with your project.

If you have any issues with this plugin, please drop an issue and tag me [@Parables](https://github.com/Parables) so that I can get notified and help you out.

Enjoy,
Parables Boltnoel

Made with love 🇬🇭
